test/lib/incentive-relationships.test.ts

```diff
@@ -165,7 +165,49 @@ test('test incentive relationship and combined max value logic', async t => {
     if (['B', 'E', 'F'].includes(incentive.id)) {
       t.equal(incentive.eligible, true);
     }
-    if (['A', 'C'].includes(incentive.id)) {
+    if (['A', 'C', 'D'].includes(incentive.id)) {
```

Comment:
This is just correcting an unrelated omission, right?

Reply:
Yep, I just apparently missed it last time and added it now.